### ISSUE TITLE
TENSORGET returns only zeros on INT32

### ISSUE THREAD

**Issue by gkorland** [RedisAI/RedisAI#68]
```
127.0.0.1:6379> AI.TensorSET source FLOAT 2 2 VALUES 1 2 3 4
OK
127.0.0.1:6379>  AI.TENSORGET source VALUES
1) FLOAT
2) 1) (integer) 2
   2) (integer) 2
3) 1) "1"
   2) "2"
   3) "3"
   4) "4"

127.0.0.1:6379> AI.TensorSET source INT32 2 2 VALUES 1 2 3 4
OK
127.0.0.1:6379>  AI.TENSORGET source VALUES
1) INT32
2) 1) (integer) 2
   2) (integer) 2
3) 1) (integer) 0
   2) (integer) 0
   3) (integer) 0
   4) (integer) 0
```

**Comment by lantiga** [comment]
Should be solved by #68, test failures are unrelated. In any case, I'll now try to find a fix for those and close the issue afterwards.

**Comment by lantiga** [comment]
Solved
